Homework/HW01.hs

```diff
@@ -0,0 +1,64 @@
+module Homework.HW01 where
+
+-- Homework Url https://www.cis.upenn.edu/~cis194/spring15/hw/01-intro.pdf
+
+-- Exercise 1 -----------------------------------------
+
+-- Get the last digit from a number
+lastDigit :: Integer -> Integer
+lastDigit num = num `mod` 10
+
+-- Drop the last digit from a number
+dropLastDigit :: Integer -> Integer
+dropLastDigit num = num `div` 10 
+
+-- Exercise 2 -----------------------------------------
+
+toRevDigits :: Integer -> [Integer]
+toRevDigits num 
+    | num > 0           = lastDigit num : (toRevDigits . dropLastDigit) num
+    | otherwise         = []
+
+-- Exercise 3 -----------------------------------------
+
+-- Double every second number in a list starting on the left.
```

Comment:
The solution asks for every second digit *from the right* :smile: which makes this a bit trickier.

Reply:
the comment says from left because the numbers have already been reversed when we were making list of digits from numbers 
Eg - 123456 breaks up into [6,5,4,3,2,1] using the above function `toRevDigits`
which means that if we wanted to double every second element **from right** in 123456,
we will have to double every 2nd element **from left** in list [6,5,4,3,2,1]  which gets us the required result 
i.e [6,10,4,6,2,2]
<img width="409" alt="Screenshot 2021-01-05 at 8 34 05 PM" src="https://user-images.githubusercontent.com/30658517/103661968-5c2b6000-4f95-11eb-9695-417a978191c7.png">

Reply:
Oh right, I missed that.